lte/gateway/c/core/oai/tasks/mme_app/mme_app_detach.c

```diff
@@ -61,7 +61,7 @@
 //------------------------------------------------------------------------------
 void mme_app_send_delete_session_request(
     struct ue_mm_context_s* const ue_context_p, const ebi_t ebi,
-    const pdn_cid_t cid) {
+    const pdn_cid_t cid, const bool no_delete_tunnel) {
```

Comment:
I could not quite follow why this function is modified as it is not targeting OAI CUPS.